packages/client/types/constEnums.ts

```diff
@@ -407,8 +407,7 @@ export const enum Threshold {
   MAX_STARTER_TIER_TEAMS = 2,
   MAX_POKER_STORIES = 50,
   MAX_INTEGRATION_FETCH_TIME = 10000,
-  MIN_STICKY_TEAM_MEETING_ATTENDEES = 2,
-  MIN_STICKY_TEAM_MEETINGS = 3,
+  MIN_STICKY_TEAM_MEETING_ATTENDEES = 1,
```

Comment:
+1 Maybe rename it to `MIN_ACTIVE_TEAM_MEETING_ATTENDEES` and also `ACTIVE_TEAM_LAST_MEETING_TIMEFRAME` as it's not really a definition for sticky teams anymore